### PR TITLE
update readme and add roadmap

### PR DESCRIPTION
**Pull request by bassam**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Roadmap
+
+This document defines a high level roadmap for Rook development. The dates below are subject to change but should give a general idea of what we are planning. We use the milestone feature in Github so look there for the most up-to-date and issue plan.
```
Comment (travisn) [inline ROADMAP.md]: how about a link to the milestones page?

File: README.md [hunk 2]
```diff
@@ -1,78 +1,43 @@
...
-- [What is Rook?](#what-is-rook)
-- [Status](#status)
-- [Quickstart Guides](#quickstart-guides)
-- [Advanced Configuration and Troubleshooting](#advanced-configuration-and-troubleshooting)
-- [Building](#building)
-- [Contributing](#contributing)
-- [Contact](#contact)
-- [Licensing](#licensing)
+[![GitHub release](https://img.shields.io/github/release/rook/rook/all.svg?style=flat-square)](https://github.com/rook/rook/releases)
+[![Go Report Card](https://goreportcard.com/badge/github.com/rook/rook)](https://goreportcard.com/report/github.com/rook/rook)
```
Comment (jbw976) [inline README.md]: these are really nice additions!

File: README.md [hunk 3]
```diff
@@ -1,78 +1,43 @@
...
-
-### Using Rook
-
-Once you have a Rook cluster running, you can use the `rookctl` tool to create and manage storage as shown in the following guide:
-- [Using Rook Guide](Documentation/client.md)
+Rook is an open source orchestrator for distributed storage systems running in cloud native environments.
 
-## Advanced Configuration and Troubleshooting
+Rook turns distributed storage software into a self-managing, self-scaling, self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management by utilizing the underlying
+cloud-native container management, scheduling and/or orchestration platform.
```
Comment (jbw976) [inline README.md]: standalone is not a focus for this intro anymore?

File: README.md [hunk 4]
```diff
@@ -1,78 +1,43 @@
...
 
 ## Contact
 
 Please use the following to reach members of the community:
 
-- Email: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
+- Forums: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
 - Gitter: [rook/rook](https://gitter.im/rook/rook) for general project discussions or [rook-dev](https://gitter.im/rook/rook-dev) for development discussions.
 - Twitter: [@rook_io](https://twitter.com/rook_io)
+- Email: [info@rook.io](<info@rook.io>)
```
Comment (jbw976) [inline README.md]: this link gives me a 404, does it need to be in the form of `mailto:info@rook.io`? or maybe remove the `<` and `>`?

File: README.md [hunk 5]
```diff
@@ -1,78 +1,43 @@
...
 
-For filing bugs, suggesting improvements, or requesting new features, help us out by opening an [issue](https://github.com/rook/rook/issues).
+For filing bugs, suggesting improvements, or requesting new features, please open an [issue](https://github.com/rook/rook/issues).
 
 ## Contact
 
 Please use the following to reach members of the community:
 
-- Email: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
+- Forums: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
```
Comment (jbw976) [inline README.md]: should Gitter be first in this list, due to its overwhelming popularity compared to the google forums?

File: README.md (hunk 3 above)
Comment (bassam) [inline README.md]: not in this context.

File: README.md [hunk 6]
```diff
@@ -1,78 +1,43 @@
...
-2. [Standalone](Documentation/standalone.md)
-
-### Using Rook
-
-Once you have a Rook cluster running, you can use the `rookctl` tool to create and manage storage as shown in the following guide:
-- [Using Rook Guide](Documentation/client.md)
+Rook is an open source orchestrator for distributed storage systems running in cloud native environments.
 
-## Advanced Configuration and Troubleshooting
+Rook turns distributed storage software into a self-managing, self-scaling, self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management by utilizing the underlying
```
Comment (kokhang) [inline README.md]: self-scaling and self-healing storage services.

Comment (kokhang) [inline README.md]: ...  monitoring, and resource management and by utilizing the underlying

File: README.md (hunk 2 above)
Comment (kokhang) [inline README.md]: agree. Really nice.

**Pull request merged**